sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -854,3 +857,208 @@ test_suite(
         for b in BOOT_POLICY_VALID_CASES
     ],
 )
+
+OTP_IMAGES = [
```

Comment:
nit: Commenting here in case I forget about this, we should probably review and try to unify the OTP images we use. But for now can you use a more specific variable name, e.g. `OTP_CFGS_EXEC_DISABLED` so that we don't step on each others' feet?

Reply:
Yeah, that less-specific name made more sense in the smaller BUILD file. Done.